bin/oli/Cargo.toml

```diff
@@ -29,38 +29,13 @@ repository = "https://github.com/apache/opendal"
 rust-version = "1.75"
 version = "0.41.14"
 
-[features]
-# Enable services dashmap support
-services-dashmap = ["opendal/services-dashmap"]
-# Enable services etcd support
-services-etcd = ["opendal/services-etcd"]
-# Enable services ftp support
-services-ftp = ["opendal/services-ftp"]
-# Enable services hdfs support
-services-hdfs = ["opendal/services-hdfs"]
-# Enable services ipfs support
-services-ipfs = ["opendal/services-ipfs"]
-# Enable services memcached support
-services-memcached = ["opendal/services-memcached"]
-# Enable services mini-moka support
-services-mini-moka = ["opendal/services-mini-moka"]
```

Comment:
I believe we can remove all in-memory storage services, but redis and hdfs may still worth to support.

Reply:
This is an action that we don't need feature flags for CLI which is almost used as a binary.

I'll change the related features enabled in a follow-up when I touch it. Generally speaking we should include most services and remove those for memory. The two considerations are (1) if our CI workflow supports it (esp. for release) (2) if it significantly affects the binary size.